bookmark-service/pom.xml

```diff
@@ -9,7 +9,7 @@
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
     <maven.compiler.target>1.8</maven.compiler.target>
     <maven.compiler.source>1.8</maven.compiler.source>
-    <quarkus.version>0.26.1</quarkus.version>
+    <quarkus.version>0.28.1</quarkus.version>
```

Comment:
I will change the quarkus version in master and rebase all branches later. Can you revert this change ?